pkg/services/notary/notary.go

```diff
@@ -303,7 +303,7 @@ func (n *Notary) PostPersist(bc blockchainer.Blockchainer, pool *mempool.Pool, b
 func (n *Notary) finalize(tx *transaction.Transaction) error {
 	acc := n.getAccount()
 	if acc == nil {
-		return errors.New("no available Notary account")
+		return errors.New("no available Notary account") // TODO: unreachable code, because all callers of `finalize` check that acc != nil
```

Comment:
If it is expected to be unreachable, we may remove it or panic.

Reply:
Fixed by panic (just in case).